### PR TITLE
Add Golang 1.13.1 to CI.

### PR DESCRIPTION
**Pull request by markusthoemmes**
1.13.1 is the current [stable version](https://golang.org/dl/#stable) of Golang.

The Knative project is adopting Golang 1.13 and its improvements, hence we need this to be able to keep up with upstream and run it through CI.

### CODE REVIEW

File: projects/origin-release/golang-1.13/golang.spec [hunk 1]
```diff
@@ -0,0 +1,21 @@
+Name:		golang
+Version:	v1.13.0
```
Comment (matzew) [inline projects/origin-release/golang-1.13/golang.spec]: 1.13.0 ? PR says `1.13.1`

Comment (markusthoemmes) [reply projects/origin-release/golang-1.13/golang.spec]: Compare with the 1.12 file. This is for building fake RPMs only and it has been kept at 1.12.0 as well although the version is actually at 1.12.9 now.

Comment (matzew) [reply projects/origin-release/golang-1.13/golang.spec]: ah... does not make sense, but ok :-)

**Pull request merged**